hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -381,7 +381,6 @@ object DataSourceWriteOptions {
   val DEFAULT_ASYNC_COMPACT_ENABLE_OPT_VAL = "true"
 
   // Avro Kafka Source configs
-  val KAFKA_AVRO_VALUE_DESERIALIZER = "hoodie.deltastreamer.source.kafka.value.deserializer.class"
-  // Schema to be used in custom kakfa deserializer
-  val KAFKA_AVRO_VALUE_DESERIALIZER_SCHEMA = "hoodie.deltastreamer.source.kafka.value.deserializer.schema"
+  val KAFKA_AVRO_VALUE_DESERIALIZER_OPT_KEY = "hoodie.deltastreamer.source.kafka.value.deserializer.class"
```

Comment:
nit: can we reuse the property prefix variable here as well? KAFKA_AVRO_VALUE_DESERIALIZER_PROPERTY_PREFIX

Reply:
I also thought of that but I was unable to import the hudi-utilities Class in the hudi-spark module I think. It would be great if you could give a hint how to achieve this.

Reply:
Just tried it out again. Adding hudi-utilities as dependency to the pom of hudi-spark introduces a circular dependency :/